src/function/scalar/string/hex.cpp

```diff
@@ -196,6 +400,23 @@ void HexFun::RegisterFunction(BuiltinFunctions &set) {
 	from_hex.name = "unhex";
 	set.AddFunction(to_hex);
 	set.AddFunction(from_hex);
+
+	ScalarFunctionSet to_binary("to_binary");
+	ScalarFunctionSet from_binary("from_binary");
+	to_binary.AddFunction(ScalarFunction({LogicalType::BIGINT}, LogicalType::VARCHAR, ToBinaryFunction));
```

Comment:
Is there a reason we don't template the ToBinaryFunction? since we already know the type here, we could get rid of the switch case at runtime, no?